modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,160 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.2.0
+
+Dear ownCloud administrator, please find, below, the changes and known issues in ownCloud Server 10.2 that need your attention. 
+You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Please note that this minor release contains database migrations which impact the upgrade duration.
+Specifically:
+
+* The `oc_share` table has a new column. The impact of this change depends on the number of shares in your ownCloud installation.
+* The `oc_authtoken` table's login name column size has been increased. The impact depends on the number of recently logged in users plus several app passwords.
```

Comment:
"plus several app passwords"? Not sure what that's supposed to mean :)

Reply:
@settermjd instead of "plus several app passwords" it could be "and the number of app passwords that have been created"